public/index.html

```diff
@@ -9,28 +9,28 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
-    
-    <img src="./images/avatar.png" alt="Icon" style="width:200px;height:200px;">
 
-    <div>
-        <h1>Friend Request</h1>
-    </div>
+<div id="opaiquebox">
+    <img id="logo" src="./images/avatar.png" alt="Icon">
 
-    <div>
+    <div id="info">
+        <h1>Friend Request</h1>
         <p>This app is an aggregator of you and your friends social media accounts, allowing you to connect with eachother easily over the many platforms you use.</p>
-        <p><a href='https://play.google.com/store/apps?hl=en'>Android Play Store</a></p>
-        <p><a href='https://www.apple.com/ca/ios/app-store/'>iOS App Store</a></p>
-        <br>
         <p>GitHub links:</p>
-        <p><a href="https://github.com/benstraats/Friend-Request-API">API</a></p>
-        <p><a href="https://github.com/benstraats/Friend-Request-React-Native">React Native</a></p>
-        <p><a href="https://github.com/benstraats/Friend-Request-Website">Web Front End</a></p>
-        <p><a href="https://github.com/benstraats/Friend-Request-Machine-Learning">Machine Learning</a></p>
-        <p><a href="https://github.com/benstraats/Friend-Request-React-Native">Android App (Deprecated)</a></p>
-        <br>
-        <br>
-        <p>App created by Ben Straatsma, Ori Wiegner, and Niko Victorino</p>
+        <p><a class="links" href="https://github.com/benstraats/Friend-Request-API">API</a></p>
+        <p><a class="links" href="https://github.com/benstraats/Friend-Request-React-Native">React Native</a></p>
+        <p><a class="links" href="https://github.com/benstraats/Friend-Request-Website">Web Front End</a></p>
+        <p><a class="links" href="https://github.com/benstraats/Friend-Request-Machine-Learning">Machine Learning</a></p>
+        <p><a class="links" href="https://github.com/benstraats/Friend-Request-React-Native">Android App (Deprecated)</a></p>
+        <div>
+          <p><a href='https://play.google.com/store/apps?hl=en'><img class="Icon1"src="./images/AIcon.png" alt="andriodicon"></a></p>
+          <p><a href='https://www.apple.com/ca/ios/app-store/'><img class="Icon2"src="./images/ios.png" alt="appleicon"></a></p>
```

Comment:
can you make these links before the GitHub links? And also say something like download the app here. And also make them side my side?